support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -44,16 +42,6 @@ class CustomActionBuilders(
         byteString
       })
       loggedAccumulator
-        .map { result =>
```

Comment:
I don't think this will catch JSON deserialisation errors any more if we do this change.

I think json deserialisation error is a 4xx but it occurs at the circe level rather than in the application logic, hence why I had to put this block in at the Play level.  I think one of the original issues was a gclid serialisation issue with guardian weekly, hence the need for this particular approach.

Might be worth testing that out by POSTing an invalid json to CODE and seeing what the status code is.

Reply:
Good idea I'll try that

Reply:
Ok I've changed this now so that we still alarm in the action but 400s from the controller will have the `result.header.reasonPhrase` value set to the error code allowing us to filter out ones from disallowed emails